R/merge_sce_list.R

```diff
@@ -38,11 +38,25 @@
 #'  value is `cell_id`.
 #' @param include_altexp Boolean for whether or not any present alternative experiments
 #'  should be included in the final merged object. Default is TRUE.
+#' @param retain_altexp_coldata_cols Named list containing vectors of column names
+#'  that should be retained in alternative experiment colData. Elements are named by the altExp
+#'  for which the columns should be retained. Default is `NULL`.
 #'
 #' @return A SingleCellExperiment object containing all SingleCellExperiment objects
 #'   present in the inputted list
 #' @export
 #'
+#' @examples
+#' \dontrun{
+#' merge_sce_list(
+#'   sce_list = list("sce1" = sce1, "sce2" = sce2),
+#'   retain_altexp_coldata_cols = list(
+#'     "adt" = c("discard", "high.controls"),
+#'     "other_altexp" = c("first_column", "second_column")
+#'   )
+#' )
```

Comment:
I think I would have more than one example here. Maybe one with altExps and one without? You can use comments (within comments 🤯)here to describe what each example is meant to demonstrate.

See https://github.com/AlexsLemonade/scpcaTools/blob/main/R/read_alevin.R#L32C1-L51C5 for an example example

Reply:
Eh? https://github.com/AlexsLemonade/scpcaTools/pull/261/commits/da703d830869fa12ef95f4db3d24d42e68cbe4b3

Reply:
I would flip the order here, probably, but the main thing I would want to see is an explanation of what the structure of the SCE going into the altExp example is: Explicitly say in the comment that these are SCEs that may contain altExpName `adt` and that those are expected to have a couple of columns. 

While I am looking at it, it is probably worth noting in the argument explanation (and maybe again in the example comment) that any named values that are not present (i.e. if there is no `adt` altExp in the SCE) that these will be ignored.